src/Commands/ProjectCommand.php

```diff
@@ -102,18 +102,29 @@ public function updateProject(
         $configuration = file_get_contents($configurationPath);
 
         if (!$configuration) {
-            $io->write(sprintf('Configuration not found on path : %s', $configurationPath));
+            $io->writeln(sprintf('Configuration not found on path : %s', $configurationPath));
             throw new InvalidArgumentException();
         }
 
         try {
             $configuration = json_decode($configuration, true);
         } catch (InvalidArgumentException $exception) {
-            $io->write('Configuration is not valid JSON ');
+            $io->writeln('<error>Configuration is not valid JSON<error>');
             throw $exception;
         }
 
-        Project::update($projectId, $configuration);
+        // Multiple projects.
+        if($projectId < 1 ) {
+            foreach($configuration as $projectId => $project_config) {
+                Project::update($projectId, $project_config);
+                $io->writeln('Project <info>' . $projectId . '</info> updated.' );
+            }
+        }
+        else {
+            // Single project.
+            Project::update($projectId, $configuration);
+            $io->writeln('Project <info>' . $projectId . '</info> updated.' );
+        }
```

Comment:
Hello @julien- 
First of all, thanks for the contribution.
I like the way you modified create command, but here in update command it seems not so natural. Having 0 (or < 1) from console in case of multiple projects creates ambiguity of behavior. I think for update we can have two separate commands: for one project (`diffy project:update id ./path`) and for multiple (`diffy projects:update ./path`). In this case users will receive clear and predictable behavior.

What do you think? Could you implement changes?

Reply:
Dear @gregurco ,

BIG thanks to you for this tool.
I updated the code and centralized the 20 lines copy & paste to do the same thing (verify an input JSON config).

If you have changes to do, be free to manage it. I finished my task to do on Diffy so I can not work more on this.

Thanks for the work to all developers on this project